index.html

```diff
@@ -523,36 +515,6 @@ <h2 style="text-align: center;">Our Newsletter</h2>
   <!--==========================
     Footer
   ============================-->
-  <footer id="footer">
-    <div class="container">
-    	 <div class="credits">
-    	 	<div class="col-3 pull-left">
-    	 	<a href="">Report a Problem</a>
-    	 		</div>
-    	 		<div class="col-3 pull-right">
-    	 			<span class="followus">Follow us on</span>
-    	 			<a href="" class="instagram">
-    	 				<i class="fa fa-instagram">
-    	 			</i>
-    	 			</a>
-    	 			<a href="" class="facebook">
-    	 				<i class="fa fa-facebook">
-    	 				</i>
-    	 		</a>
-    	 		</div>
-        <!--
-          All the links in the footer should remain intact.
-          You can delete the links only if you purchased the pro version.
-          Licensing information: https://bootstrapmade.com/license/
-          Purchase the pro version with working PHP/AJAX contact form: https://bootstrapmade.com/buy/?theme=BizPage
-        -->
-      </div>
-      <div class="copyright">
-        &copy; 2018 <strong>ConnectEd</strong>. All Rights Reserved
-      </div>
-     
-    </div>
-  </footer><!-- #footer -->
 
```

Comment:
`  <footer id="footer">
-    <div class="container">
-    	 <div class="credits">
-    	 	<div class="col-3 pull-left">
-    	 	<a href="">Report a Problem</a>
-    	 		</div>
-    	 		<div class="col-3 pull-right">
-    	 			<span class="followus">Follow us on</span>
-    	 			<a href="" class="instagram">
-    	 				<i class="fa fa-instagram">
-    	 			</i>
-    	 			</a>
-    	 			<a href="" class="facebook">
-    	 				<i class="fa fa-facebook">
-    	 				</i>
-    	 		</a>
-    	 		</div>
-        <!--
-          All the links in the footer should remain intact.
-          You can delete the links only if you purchased the pro version.
-          Licensing information: https://bootstrapmade.com/license/
-          Purchase the pro version with working PHP/AJAX contact form: https://bootstrapmade.com/buy/?theme=BizPage
-        -->
-      </div>
-      <div class="copyright">
-        &copy; 2018 <strong>ConnectEd</strong>. All Rights Reserved
-      </div>
-     
  </div>
 </footer><!-- #footer -->`